common/logisticspipes/pipes/PipeFluidTerminus.java

```diff
@@ -0,0 +1,107 @@
+package logisticspipes.pipes;
+
+import logisticspipes.LogisticsPipes;
+import logisticspipes.interfaces.ITankUtil;
+import logisticspipes.interfaces.routing.IFluidSink;
+import logisticspipes.network.GuiIDs;
+import logisticspipes.pipes.basic.fluid.FluidRoutedPipe;
+import logisticspipes.textures.Textures;
+import logisticspipes.transport.PipeFluidTransportLogistics;
+import logisticspipes.utils.*;
+import logisticspipes.utils.item.ItemIdentifierInventory;
+import logisticspipes.utils.tuples.Triplet;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.fluids.FluidTank;
+
+public class PipeFluidTerminus extends FluidRoutedPipe implements IFluidSink {
+    public ItemIdentifierInventory filterInv = new ItemIdentifierInventory(9, "Dummy", 1, true);
```

Comment:
Make it final; use properties, i.e. `InventoryProperty` and use `PropertyLayer` on the GUI. Have a look at the `ModuleProvider` & provider GUI classes to check how to use them.

Reply:
I dont think this is worth the effort of copying the whole property system over from the module implementation for just a single pipe. Works fine without PropertyLayer.

Please explain, for my understanding, what the benefits of using Property Layer and everything that comes with it instead of the method used in the other fluid pipes.

Reply:
Sure, we are switching the whole GUI system to a observable pattern. The `InventoryProperty` is a drop-in replacement, so even if you cannot update the GUI, please at least use the property in the pipe.

Reply:
Ill definitly drop that in then. And perhaps take a stab at the UI and the stuff around it later.

Reply:
I updated the basic and terminus pipes to use Properties for now. The UI stuff requires changes that go far deeper than I am comfortable with. I would rather not touch CoreRoutedPipe for now.